oak/server/oak_node.h

```diff
@@ -70,17 +62,19 @@ class OakNode {
   bool WaitOnChannels(std::vector<std::unique_ptr<ChannelStatus>>* statuses) const;
 
  protected:
+  // If the Node has a single registered handle, return it; otherwise, return
```

Comment:
What does `registered handle` mean? Does this not include handles that are passed in from other nodes? Also isn't the single handle usually just for input? maybe it should be called `InputHandle`?

Reply:
"registered handle" here is just a handle in the `channel_halves_` table; this is a convenience for Node startup, when there should be exactly one entry in the table (and the handle for it gets passed to `oak_main`).

Expanded the comment.

Reply:
I still think it may make sense at some point to give a special name (or alias) to the initial input channel, kind of like `stdin`. That way a method that returns it is guaranteed to always return a valid handle, and in fact always the same handle even if called later on during the execution of the node. Anyway, perhaps something for another PR, if you think it's useful.

Reply:
I guess we can come back to that if it's needed.  It shouldn't be hard to save off the initial parameter value passed to `oak_main` though, so having a host function for it seems a wee bit unnecessary.